index.html

```diff
@@ -15,22 +15,280 @@
+                <li>HTML</li>
+              </ul>
+
+              <button type="button" class="btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section id="about" class="flex-column container center-align">
+          <h2 class="bold big-font center-text">About</h2>
+          <p class="regular regular-font center-text about-desc">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <button type="button" class="btn">Get My Resume</button>
+
+        <div class="work-grid">
+          <div class="grid-item about-item">
+            <div class="about-icons">
+              <img src="assets/img/lang.svg" alt="Languages Icon" />
+            </div>
+
+            <h3 class="semi-bold center-text work-title">Languages</h3>
+
+            <ul class="language flex-row about-lang color2">
+              <li>Ruby</li>
+              <li>Css</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+          </div>
+
+          <div class="grid-item about-item">
+            <div class="about-icons">
+              <img src="assets/img/frame.svg" alt="Languages Icon" />
+            </div>
+
+            <h3 class="semi-bold center-text work-title">Frameworks</h3>
+
+            <ul class="language flex-row about-lang color2">
+              <li>Ruby</li>
+              <li>Css</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+          </div>
+
+          <div class="grid-item about-item">
+            <div class="about-icons">
+              <img src="assets/img/skills.svg" alt="Languages Icon" />
+            </div>
+
+            <h3 class="semi-bold center-text work-title">Skills</h3>
+
+            <ul class="language flex-row about-lang color2">
+              <li>Ruby</li>
+              <li>Css</li>
+              <li>Javascript</li>
+              <li>HTML</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section id="contact" class="flex-column container center-align">
+        <h2 class="bold big-font center-text color1">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+
+        <form action="#" id="contact-form" class="flex-column center-align">
+          <input type="text" name="name" placeholder="Full name" />
+          <input type="email" name="email" placeholder="Email address" />
+          <textarea name="comment" id="comment" cols="30" rows="10">
+Enter text here</textarea
+          >
+          <button type="submit" class="btn">Get in touch</button>
+        </form>
+      </section>
+    </main>
+
+    <footer class="flex-column container center-align">
```

Comment:
- [ ] Nice to add a contact section but this is not in this milestone. Please add it to the next milestone in another branch, so another TSE can review it. For that, you can just comment on this contact section.

- Comment all part after skills section.

![image](https://user-images.githubusercontent.com/38635158/164805492-77138d29-8bf7-4e23-b62d-49295771dd6a.png)